### ISSUE TITLE
[Question]: Can I use AWS BedRock with Fabric ?

### ISSUE THREAD

**Issue by msbaek** [danielmiessler/Fabric#1193]
### What is your question?

I was wondering if there is a way to use amazon bedrock in fabric.
I can't find any clue in documenation.
Any help will be welcomed
thanks.

**Comment by kvorion** [comment]
I had the same question. I'm surprised there's nothing in the documentation about which models are supported.

**Comment by mattjoyce** [comment]
You can see the supported platforms here : https://github.com/danielmiessler/fabric/tree/main/plugins/ai

**Comment by ksylvan** [comment]
Yes, as of around [v1.4.203](https://github.com/danielmiessler/Fabric/blob/main/CHANGELOG.md#v14203-2025-06-14) we have support for Bedrock.
